src/agent/src/storage/mod.rs

```diff
@@ -380,8 +381,122 @@ fn mount_storage(logger: &Logger, storage: &Storage) -> Result<()> {
         "mount-options" => options.as_str(),
     );
 
+    let confidential = storage.driver_options.contains(&"confidential=true".into());
+    let ephemeral = storage.driver_options.contains(&"ephemeral=true".into());
+
+    let src_path = if confidential && ephemeral {
```

Comment:
This if branch is too long. Would it be practical to move all/most of the if branch code into a separate function?

Reply:
Done